make/autoconf/spec.gmk.in

```diff
@@ -330,6 +330,8 @@ USE_PRECOMPILED_HEADER := @USE_PRECOMPILED_HEADER@
 # Only build headless support or not
 ENABLE_HEADLESS_ONLY := @ENABLE_HEADLESS_ONLY@
 
+ENABLE_CONFORMING_COMPILATION := @ENABLE_CONFORMING_COMPILATION@
```

Comment:
You don't need to export this in spec.gmk, since it is only checked in the configure script. We try hard to only export here what is strictly necessary for use by makefiles. If you will ever need it in the future, please add it as an export here at that time.

Reply:
Alright, will do